lib/utils.js

```diff
@@ -64,7 +64,11 @@ function getHookValue(data, job, key) {
 }
 
 function makeHook(format, job) {
-  format = format || DEFAULT_FORMAT
+  try {
+    format = JSON.parse(format)
```

Comment:
`format` should not be a string. It should be an object because `crawlTree` function iterates through an object.
I am using `try-catch` block here to parse format from string to object. If parsing fails, `format` gets its default value.